app/src/main/java/ca/bc/gov/bchealth/ui/component/HealthRecordItemUI.kt

```diff
@@ -0,0 +1,111 @@
+package ca.bc.gov.bchealth.ui.component
+
+import androidx.annotation.DrawableRes
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material.Card
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import ca.bc.gov.bchealth.R
+import ca.bc.gov.bchealth.compose.BasePreview
+import ca.bc.gov.bchealth.compose.MyHealthTheme
+import ca.bc.gov.bchealth.compose.blue
+import ca.bc.gov.bchealth.compose.grey
+import ca.bc.gov.bchealth.compose.lightGrey
+import ca.bc.gov.bchealth.compose.smallButton
+
+@Composable
+fun HealthRecordItemUI(
+    @DrawableRes image: Int,
+    title: String,
+    description: String,
+    onClick: () -> Unit
+) {
+    Card(
+        modifier = Modifier
+            .clickable { onClick() }
+            .fillMaxWidth(),
+        shape = MaterialTheme.shapes.medium,
+        backgroundColor = lightGrey,
+        elevation = 0.dp,
+    ) {
+
+        ConstraintLayout(
+            modifier = Modifier
+                .padding(16.dp)
+                .fillMaxWidth()
+        ) {
+            val (icon, txtTitle, txtDescription, rightArrowImage) = createRefs()
+            Image(
+                modifier = Modifier.constrainAs(icon) {
+                    start.linkTo(parent.start)
+                    top.linkTo(parent.top)
+                },
+                painter = painterResource(id = image),
+                contentDescription = stringResource(id = R.string.notes_content_description)
```

Comment:
I noted that the same content description is used in the image below, are them required? If it's merely decorative you can suppress it

Reply:
I will update this, as part of my next PR.